packages/support/resources/views/components/input/select.blade.php

```diff
@@ -21,7 +21,7 @@
     @endif
     {{
         $attributes->class([
-            'fi-select-input block w-full border-none bg-transparent py-1.5 pe-8 text-base transition duration-75 focus:ring-0 disabled:text-gray-500 disabled:[-webkit-text-fill-color:theme(colors.gray.500)] dark:disabled:text-gray-400 dark:disabled:[-webkit-text-fill-color:theme(colors.gray.400)] sm:text-sm sm:leading-6',
+            'fi-select-input block w-full border-none bg-transparent py-2 pe-8 text-base transition duration-75 focus:ring-0 disabled:text-gray-500 disabled:[-webkit-text-fill-color:theme(colors.gray.500)] dark:disabled:text-gray-400 dark:disabled:[-webkit-text-fill-color:theme(colors.gray.400)] sm:text-sm sm:leading-6',
```

Comment:
@awcodes, what does this fix exactly?
I'm pretty sure it should be `py-1.5`.

Reply:
Selects don't have a line height so the padding on them always has to be more that on a standard input. The screenshots show the difference.

Reply:
Hmm, could you share the height in `px` after this change?

Reply:
Oh just saw your other comment.